### DISCUSSION TITLE
Communicating with juicefs minio using docker volume

### DISCUSSION THREAD

**Discussion by Hyeyeo-Kim** [juicedata/juicefs#5293] [Q&A]
I am using minio as a container. I want to use it by communicating with juicefs and minio, so I am creating a container by looking at the juicefs guide. However, communication is currently impossible. Below, I will give you the method and code I used to install juicefs. Could you help me?

Now minio is working well

1. Install docker juicefs plugin
    docker plugin install juicedata/juicefs

2. Create juicefs volume
    docker volume create -d juicedata/juicefs \
    -o name=jfsvolue \
    -o metaurl="s3://ip:port/test" \
    -o storage="s3" \
    -o bucket=test \
    -o access-key="key" \
    -o secret-key="key" \
    jfsvolume

3. Create container
    docker run -it --rm \
    -v jfsvolume:/data-shared \
    ubuntu:latest

$ docker volume ls
DRIVER                     VOLUME NAME
juicedata/juicefs:latest   jfsvolume

$ docker volume inspect jfsvolume
[
    {
        "CreatedAt": "0001-01-01T00:00:00Z",
        "Driver": "juicedata/juicefs:latest",
        "Labels": {},
        "Mountpoint": "/jfs/volumes/jfsvolue",
        "Name": "jfsvolume",
        "Options": {
            "access-key": "key",
            "bucket": "s3://ip:port/test",
            "metaurl": "s3://ip:port/test",
            "name": "jfsvolue",
            "secret-key": "key",
            "storage": "s3"
        },
        "Scope": "local"
    }
]

**Comment by zxh326** [comment]
need more logs

https://juicefs.com/docs/community/juicefs_on_docker/#troubleshooting
